lua/hop/keymap.lua

```diff
@@ -17,7 +17,7 @@ function M.create_jump_keymap(buf_handle, opts)
     vim.api.nvim_buf_set_keymap(buf_handle, '', key, "<cmd>lua require'hop'.refine_hints(0, '" .. key .. "')<cr>", { nowait = true })
   end
 
-  vim.api.nvim_buf_set_keymap(buf_handle, '', '<esc>', '<cmd>q<cr>', {})
+  vim.api.nvim_buf_set_keymap(buf_handle, '', '<esc>', '<cmd>close<cr>', {})
```

Comment:
What’s the difference between `close` and `quit` here?

Reply:
As the document says

```
							*:q* *:quit*
:q[uit]			Quit the current window.  Quit Vim if this is the last
			window(my note: not including help window and preview window). 
                        This fails when changes have been made and
			Vim refuses to |abandon| the current buffer, and when
			the last file in the argument list has not been
			edited.
```

If the `help` window is the only window in Vim, then after using `HopWord`, the hop window will be the last window, in that case if using `:q` to quit hop window, vim will exit.